web-services/query/src/main/java/datawave/webservice/query/iterator/DatawaveTransformIterator.java

```diff
@@ -1,13 +1,13 @@
 package datawave.webservice.query.iterator;
 
+import datawave.webservice.query.exception.EmptyObjectException;
 import datawave.webservice.query.logic.Flushable;
-import java.util.Iterator;
-
 import org.apache.commons.collections4.Transformer;
```

Comment:
Please revert this class to what is in integration.

Reply:
performed `git checkout integration ./web-services/query/src/main/java/datawave/webservice/query/iterator/DatawaveTransformIterator.java`